plugins/c9.vfs.client/vfs_client.js

```diff
@@ -193,12 +193,12 @@ define(function(require, exports, module) {
                 extraPaths = path;
                 path = path[0];
                 extraPaths = "," + extraPaths.map(function(p) {
-                    return p[0] == path[0] && p != path ? escape(p) : "";
+                    return p[0] == path[0] && p != path ? encodeURI(p) : "";
```

Comment:
for which filenames escape was giving wrong results?
Asking because without `escape` filenames with `,` will be interpreted as two different filenames also vfsUrl bellow still uses escape.